### PR TITLE
Updates to micro art

### PR DESCRIPTION
**Pull request by mmcfarland**
* Insert missing LC type images
* Fix styling bugs (h/t @jfrankl )

Connects #865 
Connects #766

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now.

**Comment by rajadain** [comment]
+1 for functionality. Regarding the art, all the tiles have a slight shade on the left side and a slight tint on the right, including the new Wetlands:

image

But the Open Water tile does not, and looks odd:

image

It also does not have a vertical edge in the center.

Tested on regular and Retina screens, and the images look good. Did discover an issue when testing on an iPad: #904, likely caused by the latest iOS 9, but that can be tackled separately.

File: src/mmw/sass/pages/_water-balance.scss [hunk 1]
```diff
@@ -300,9 +300,13 @@
         float: left;
         position: relative;
         margin: 8px 8px 0 8px;
-        width: 66px;
+        width: 61px;
         white-space: normal;
 
+        &:nth-child(4n+5) {
```
Comment (rajadain) [inline src/mmw/sass/pages/_water-balance.scss]: This is really clever.

File: src/mmw/sass/pages/_water-balance.scss [hunk 2]
```diff
@@ -300,9 +300,13 @@
         float: left;
         position: relative;
         margin: 8px 8px 0 8px;
-        width: 66px;
+        width: 61px;
```
Comment (rajadain) [inline src/mmw/sass/pages/_water-balance.scss]: So did we accidentally increase the bounding box size or decrease the tile size necessitating this?

**Comment by rajadain** [comment]
Nice job, very clean!

**Comment by mmcfarland** [comment]
@designmatty - could you take a look at open water image as @rajadain described?  Seems like a quick touchup could bring it in line with the existing artwork.

**Comment by mmcfarland** [comment]
Bounced the image updates to a new issue https://github.com/WikiWatershed/model-my-watershed/issues/907 so I can merge this.

**Pull request merged**